types/src/on_chain_config/aptos_features.rs

```diff
@@ -11,6 +11,7 @@ pub enum FeatureFlag {
     CODE_DEPENDENCY_CHECK = 1,
     TREAT_FRIEND_AS_PRIVATE = 2,
     VM_BINARY_FORMAT_V6 = 5,
+    COLLECT_AND_DISTRIBUTE_GAS_FEES = 6,
```

Comment:
Can you turn this feature on by default in genesis so devnet would have it by default and we can test it there? See https://github.com/aptos-labs/aptos-core/pull/6040/files#diff-c357acdc9a799c00c416262b0fd8e627e1f882f603e63d77625ae0f0fd9535faR402 for example

cc @sherry-x

Reply:
Actually, we cannot. Because then we have to initialise the resources to collect the fees as well.